params/config.go

```diff
@@ -45,13 +45,15 @@ var (
 		LondonCompatibleBlock:    big.NewInt(86816005),
 		EthTxTypeCompatibleBlock: big.NewInt(86816005),
 		MagmaCompatibleBlock:     big.NewInt(99841497),
+		KoreCompatibleBlock:      big.NewInt(0xffffffff),
 		DeriveShaImpl:            2,
 		Governance: &GovernanceConfig{
 			GoverningNode:  common.HexToAddress("0x52d41ca72af615a1ac3301b0a93efa222ecc7541"),
 			GovernanceMode: "single",
 			Reward: &RewardConfig{
 				MintingAmount:          mintingAmount,
 				Ratio:                  "34/54/12",
+				Kip82Ratio:             "20/80",
```

Comment:
Perhaps we shouldn't add new parameters to `CypressChainConfig` and `BaobabChainConfig`?

Reply:
@aidan-kwon @kjhman21 What is the policy of `CypressChainConfig`? Should new params be initialized here or not?
It will work either way, whether it has been initialized like this line, or not been initialized like KIP71 params.
If the former, we can add KIP71 params initializations. Otherwise, we can remove this line.

Reply:
@blukat29 The lines are removed.